packages/notifi-react-example/src/NotifiCard/SolanaCard.tsx

```diff
@@ -55,137 +56,35 @@ export const SolanaCard: React.FC = () => {
                 </li>
               ))}
           </ul>
-          <h3>Notification history</h3>
-          <button
-            onClick={() =>
-              client
-                .getUnreadNotificationHistoryCount()
-                .then((res) => alert(JSON.stringify(res)))
-            }
-          >
-            1. fetch unread notification history count
```

Comment:
Client operations' example should only in `frontend-client` example. `Card` just simple as it is.